src/App/Game/GameModePicker.tsx

```diff
@@ -144,6 +152,34 @@ export class GameModePicker extends React.Component<IGameModePickerProps, IGameM
             />
           </div>
         );
+      } else if (extraInfo.type === 'dropdown') {
+        const dropdown = (extraInfo as IGameModeExtraInfoDropdown);
+        const list: JSX.Element[] = [];
+        dropdown.options.map((option, idx) => {
```

Comment:
Map converts each element of a list into another element of a new list. So you should not use a side effect such push inside map.

Do list = dropdown.options.map yada yada yada return <MenuItem

Reply:
Good idea, thanks!